commons-hpcc/src/main/java/org/hpccsystems/commons/ecl/RecordDefinitionTranslator.java

```diff
@@ -137,6 +141,12 @@ private static boolean isUnsigned(int typeID)
         return (typeID & FLAG_UNSIGNED) != 0;
     }
 
+    private static int getAdditionalFlags(int flags)
```

Comment:
Is this method for callers interested only in the 0x8000 bit, or are would they also be interested in the higher bits